parsec/backend/memory/factory.py

```diff
@@ -22,7 +25,9 @@
 
 @asynccontextmanager
 async def components_factory(config: BackendConfig, event_bus: EventBus):
-    (send_events_channel, receive_events_channel) = trio.open_memory_channel(math.inf)
+    (send_events_channel, receive_events_channel) = trio.open_memory_channel[
```

Comment:
the parenthesis around `(send_events_channel, receive_events_channel) =` was probably added by formatting, I think we should remove it to save a bit more space

Reply:
> to save a bit more space

"Y'a pas de petits profits"

Reply:
9ddb38c5be108a3fc89d6ff4ab2404d92401c0ed